plugins/hls-retrie-plugin/src/Ide/Plugin/Retrie.hs

```diff
@@ -416,12 +421,19 @@ callRetrie state session rewrites origin restrictToOriginatingFile = do
       (theImports, theRewrites) = partitionEithers rewrites
 
       annotatedImports =
-        unsafeMkA (map (GHC.noLoc . toImportDecl) theImports) mempty 0
+#if MIN_VERSION_ghc(9,2,0)
+        unsafeMkA (map (noLocA . toImportDecl) theImports) 0
+#else
+        unsafeMkA (map (noLocA . toImportDecl) theImports) mempty 0
+#endif
 
   (originFixities, originParsedModule) <- reuseParsedModule origin
   retrie <-
     (\specs -> apply specs >> addImports annotatedImports)
       <$> parseRewriteSpecs
+#if MIN_VERSION_ghc(9,2,0)
+        libdir -- TODO: does this actualy get the proper libdir?
+#endif
```

Comment:
@pepeiborra this is the biggest question mark I have. I assume this should point to a user's libdir?

Reply:
This will get the HLS libdir, which will probably not work since it will point to a location in the build agent that compiled HLS...

Reply:
To get the user libdir, grab a GHC session and use https://hackage.haskell.org/package/ghc-9.4.2/docs/GHC-Driver-Session.html#v:topDir

Reply:
Yeah thought as much, didn't know if ghc-paths used some trickery to get the actual path.